course/frontend/arrays/scripts/main.js

```diff
@@ -213,3 +213,80 @@ function flat(arr, newArray) {
 
 flat([1, 2, 3, [4, 5], 6]);
 flat([1, 2, [3, 4, [1, 2, [1, 2, 3]]], 5, 6]);
+
+
+// Exercises 28, 29
+
+/*
+### unionArray
+
+Write a JavaScript program to compute the union of two arrays.
+Sample Data :
+
+    console.log(unionArray([1, 2, 3], [100, 2, 1, 10]));
+    [1, 2, 3, 10, 100]
+
+### removeFalsy
```

Comment:
MOVE this "requirement / explanation" about "removeFalsy" just above its doc / comment ,)